pkgs/development/beam-modules/default.nix

```diff
@@ -64,8 +64,9 @@ let
                        debugInfo = true;
                      };
 
-        lfe = lfe_1_2;
+        lfe = lfe_1_3;
         lfe_1_2 = lib.callLFE ../interpreters/lfe/1.2.nix { inherit erlang buildRebar3 buildHex; };
+        lfe_1_3 = lib.callLFE ../interpreters/lfe/1.3.nix { inherit erlang buildRebar3 buildHex; };
```

Comment:
Do we need both lfe_1_2 and 1.3? The rule of thumb is, that we usually do this for projects where multiple maintained major versions exists. This seems not the case here.

Reply:
see Q+A above :smiley: